src/extension/inspire/src/main/java/org/geoserver/inspire/wcs/WCSExtendedCapabilitiesProvider.java

```diff
@@ -83,19 +84,7 @@ public void encodeExtendedContents(
             tx.end("inspire_common:MediaType");
         }
         tx.end("inspire_common:MetadataUrl");
-        tx.start("inspire_common:SupportedLanguages");
-        language = language != null ? language : "eng";
-        tx.start("inspire_common:DefaultLanguage");
-        tx.start("inspire_common:Language");
-        tx.chars(language);
-        tx.end("inspire_common:Language");
-        tx.end("inspire_common:DefaultLanguage");
-        tx.end("inspire_common:SupportedLanguages");
-        tx.start("inspire_common:ResponseLanguage");
-        tx.start("inspire_common:Language");
-        tx.chars(language);
-        tx.end("inspire_common:Language");
-        tx.end("inspire_common:ResponseLanguage");
+        ViewServicesUtils.encodeSupportedLanguages(language,tx);
```

Comment:
Confusing though, WCS is not a view service, it's a download service.

Reply:
fixed renamed ViewServiceUtils to ServiceUtils